crankshaft-engine/src/service/runner/backend/docker.rs

```diff
@@ -98,7 +98,7 @@ impl crate::Backend for Backend {
                 // (2) Upload inputs to the container.
                 //
                 // TODO(clay): these could be cached.
-                for task in task.inputs().cloned().map(|i| {
+                for task in task.inputs().map(|i| {
```

Comment:
As you'll see, `inputs` now just returns an `Arc<Input>` instead of `&Arc<Input>`.